api/client/network.go

```diff
@@ -0,0 +1,9 @@
+package client
+
+import nwclient "github.com/docker/libnetwork/client"
+
+func (cli *DockerCli) CmdNetwork(args ...string) error {
+	nCli := nwclient.NewNetworkCli(cli.out, cli.err, nwclient.CallFunc(cli.call))
+	args = append([]string{"network"}, args...)
+	return nCli.Cmd("docker", args...)
```

Comment:
Ah, ok... so you want "docker" to display in the help output.
Should this be `os.Args[0]` instead?